qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -274,58 +349,36 @@ def _build_hopping_ops(
                 f"type {type(problem)}"
             )
 
-    def _build_all_commutators(
-        self, hopping_operators: dict, type_of_commutativities: dict, size: int
+    def _build_all_eom_operators(
+        self,
+        pre_tap_operator: PauliSumOp,
+        expansion_basis_data: Tuple[Dict[str, PauliSumOp], Dict[str, List[bool]], int],
     ) -> dict:
         """Building all commutators for Q, W, M, V matrices.
 
         Args:
-            hopping_operators: All hopping operators based on excitations_list,
+            pre_tap_operator: Pre tapered Hamiltonian operator
```

Comment:
```suggestion
            pre_tap_operator: The untapered Hamiltonian operator.
```

Reply:
Maybe you can update the variable names to say `untap` instead of `pre_tap` :thinking:

Reply:
Indeed this naming is not ideal: the three step currently are uncap, pretap, tap. THis is due to the splitting of the tapering into two parts